src/main/java/frc/robot/commands/scoring/intake/IntakeGamePiece.java

```diff
@@ -84,7 +84,7 @@ public boolean isFinished() {
         Blinkin.setColor(Colors.VIOLET);
       }
     } else {
-      Blinkin.setColor(Colors.RED);
+      Blinkin.blinkTeamColors();
```

Comment:
I think it would be good for it to blink yellow if it is intaking a cone and purple if it is intaking a cube.  So just an if statement like `if (m_type == GamePieceType.CONE)` and then `setDualColor()` to purple or yellow

Reply:
This could be used to alert the human player for example, or just to verify it's intaking the right thing

Reply:
Okay, I think I've implemented this.